app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -1487,4 +1494,29 @@ private void renderSubtitles(@Nullable final String text) {
             binding.subtitlesText.setVisibility(View.VISIBLE);
         });
     }
+
+    public boolean getServerMeetsMinimumVersion(@NonNull String minServerVersion) {
+        if (sessionRepository.getValue().getCurrentSession().getValue() == null) {
```

Comment:
Assign the session to a variable, the value might have changed after this check. (although unlikely)